### PR TITLE
Active Response Initial Workflow Add

### PR DESCRIPTION
**Pull request by jadoncm-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+This workflow is part of the MDR Active Response add-on and is required to disable users with Active Directory.
+
+# Key Features
+
+* Disable User with Active Directory
+* Send real-time updates to actions using your preferred communication methods: phone, email, Slack, and SMS.
+* Accelerate or cancel containment actions from your mobile or desktop devices via Slack.
+
+
+
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md:12]:
```suggestion
```

File: workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Description
+
+This workflow is part of the MDR Active Response add-on and is required to disable users with Active Directory.
+
+# Key Features
+
+* Disable User with Active Directory
+* Send real-time updates to actions using your preferred communication methods: phone, email, Slack, and SMS.
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md:8]:
```suggestion
* Send real-time updates to actions using your preferred communication methods: phone, email, Slack, and SMS.
```
What's the difference between phone and SMS? Is one via phone call?

File: workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Accelerate or cancel containment actions from your mobile or desktop devices via Slack.
+
+
+
+# Requirements
+
+* Active Directory LDAP
+* Python3 Script
+* Slack (ChatOps)
+* REST
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md:18]: These should be moved to Technical Details section, example:
```
## Technical Details

Plugins utilized by workflow:

|Plugin|Version|Count|
|----|----|--------|
|Check Point NGFW|2.0.0|1|

## Troubleshooting

_There is no troubleshooting information at this time_
```

File: workflows/MDR_Active_Response_Enable_User_with_Active_Directory/help.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+# Key Features
+
+* Enable User in your environment with Active Directory
+* Reverse containment actions from your mobile or desktop devices via Slack.
+* Receive real-time updates to actions in Slack.
+
+# Requirements
+
+* Active Directory LDAP
+* Slack (ChatOps)
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Enable_User_with_Active_Directory/help.md:14]:
```suggestion
* Active Directory LDAP
* Slack (ChatOps)
```

Move to technical requirements section per the above comment

File: workflows/MDR_Active_Response_Enable_User_with_Active_Directory/help.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+## Technical Details
+
+For Technical details, refer to the MDR Containment Enable User with Active Directory import instructions.
+
+## Troubleshooting
+
+_There is no troubleshooting information at this time_
+
+# Version History
+* 1.0.0 - Initial workflow
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Enable_User_with_Active_Directory/help.md:31]:
```suggestion
# Version History

* 1.0.0 - Initial workflow
```

File: workflows/MDR_Active_Response_Isolate_Sensor_with_VMware_Carbon_Black_EDR/help.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+# Requirements
+
+* Type Converter
+* REST 
+* Python 3 Script 
+* VMware Carbon Black EDR (Not required if using the Insight Agent)
+* Active Directory LDAP 
+* REST 
+* Slack (ChatOps)
+
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Isolate_Sensor_with_VMware_Carbon_Black_EDR/help.md:21]: Technical requirements section

File: workflows/MDR_Active_Response_Isolate_Sensor_with_VMware_Carbon_Black_EDR/help.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+## Technical Details
+
+For Technical details, please refer to MDR Containment Isolate Sensor with VMware Carbon Back EDR import instructions.
+
+## Troubleshooting
+
+_There is no troubleshooting information at this time_
+
+# Version History
+* 1.0.0 - Initial workflow
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Isolate_Sensor_with_VMware_Carbon_Black_EDR/help.md:38]:
```suggestion
# Version History

* 1.0.0 - Initial workflow
```

File: workflows/MDR_Active_Response_Router/help.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+# Requirements
+
+* Insight Platform Organization-level API key
+* Python3 Script 
+* Type Converter
+* REST
+* Active Directory LDAP
+* Microsoft Office 365 Email 
+* Twilio (Optional) 
+* Slack (ChatOps)
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Router/help.md:21]: Technical requirements section

File: workflows/MDR_Active_Response_Router/help.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+For Technical Details, please refer to the import MDR Active Response Asset Quarantine Workflow page.
+
+## Troubleshooting
+
+_There is no troubleshooting information at this time_
+
+# Version History
+* 1.0.0 - Initial workflow
+
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Router/help.md:40]:
```suggestion
# Version History

* 1.0.0 - Initial workflow

```

File: workflows/MDR_Active_Response_Unisolate_Sensor_with_VMware_Carbon_Black_EDR/help.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+* Unisolate Sensor with Cb Response
+* Reverse containment actions from your mobile or desktop devices via Slack.
+* Receive real-time updates to actions in Slack.
+
+# Requirements
+
+* VMware Carbon Black EDR
+* Slack (ChatOps)
+
+
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Unisolate_Sensor_with_VMware_Carbon_Black_EDR/help.md:16]:
```suggestion
```

Comment (mrinehart-r7) [reply workflows/MDR_Active_Response_Unisolate_Sensor_with_VMware_Carbon_Black_EDR/help.md:16]: Technical requirements section

File: workflows/MDR_Active_Response_Unisolate_Sensor_with_VMware_Carbon_Black_EDR/help.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+ 
+## Technical Details
+
+For Technical details, refer to the MDR Containment Unisolate Sensor with Cb Response import instructions.
+
+## Troubleshooting
+
+_There is no troubleshooting information at this time_
+
+# Version History
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Unisolate_Sensor_with_VMware_Carbon_Black_EDR/help.md:31]:
```suggestion
# Version History

```

File: workflows/MDR_Active_Response_Unquarantine_Asset_with_Insight_Agent/help.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+* Reverse containment actions from your mobile or desktop devices via Slack.
+* Receive real-time updates to actions in Slack.
+
+
+# Requirements
+
+* Python3 Script
+* Slack (ChatOps)
+* REST
+
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Unquarantine_Asset_with_Insight_Agent/help.md:17]: Technical requirements section

File: workflows/MDR_Active_Response_Unquarantine_Asset_with_Insight_Agent/help.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+ 
+## Technical Details
+
+For Technical details, refer to the MDR Containment Unquarantine Asset with Insight Agent import instructions.
+
+## Troubleshooting
+
+_There is no troubleshooting information at this time_
+
+# Version History
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Unquarantine_Asset_with_Insight_Agent/help.md:33]:
```suggestion
# Version History

```

**Comment by jschipp-r7** [comment]
@jadoncm-r7 you need to run the validators on each workflow and paste the results here. We do not know if these are syntactically valid until that is done.

```
$ icon-validate workflows/My_Workflow
```

**Comment by mrinehart-r7** [comment]
> @jadoncm-r7 you need to run the validators on each workflow and paste the results here. We do not know if these are syntactically valid until that is done.
> 
> ```
> $ icon-validate workflows/My_Workflow
> ```
@jschipp-r7 
I ran the validators against them all, fixed them, and they now pass validation.

**Comment by jschipp-r7** [comment]
Thanks a TON @mrinehart-r7!!!!

**Pull request merged**